### ISSUE TITLE
opensearch support

### ISSUE THREAD

**Issue by sebastian-ehrling** [drud/ddev#3897]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

I've created a ddev get addon for opensearch. How can it be made available by ddev get --list?

### Describe the solution you'd like

Publish sebastian-ehrling/ddev-opensearch :)

**Comment by tyler36** [comment]
@sebastian-ehrling 

The [ddev-addon-template](https://github.com/drud/ddev-addon-template) all the recommended steps under getting started.

For it to show up under `ddev get --list --all`, you need to do 2 things

1. Create a release on GitHub.
2. Add a good short description to your repo, and add the label "ddev-get". It will immediately be added to the list provided by ddev get --list --all.

**Comment by tyler36** [comment]
Thanks for contributing. I look forward to checking it out.

**Comment by sebastian-ehrling** [comment]
Hi @tyler36 
yes with the --all parameter it is available. What do I have to do to make it available without the parameter?

**Comment by rfay** [comment]
I opened issues on both of your new add-ons, congratulations! There's plenty of work to go on both, and I'm happy to help. 
* https://github.com/sebastian-ehrling/ddev-selenium-standalone/issues/1
*  https://github.com/sebastian-ehrling/ddev-selenium-standalone/issues/1

Closing this for now, great initiatives. You'll probably want to focus on one until it gets toward maturity. Happy to continue the conversation and of course you've invited to Discord, where there's n add-on channel, #ddev-get-addons. https://discord.gg/hCZFfAMc5k
